x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_list_filter_popover.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useState, useCallback, useMemo } from 'react';
+import { EuiPopover, EuiFilterButton, useGeneratedHtmlId } from '@elastic/eui';
+
+export const ActionListFilterPopover = memo(
+  ({
+    children,
+    filterName,
+    hasActiveFilters,
+    numActiveFilters,
+    numFilters,
+  }: {
+    children: React.ReactNode;
+    filterName: string;
+    hasActiveFilters: boolean;
+    numActiveFilters: number;
+    numFilters: number;
+  }) => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+    const onButtonClick = useCallback(() => {
+      setIsPopoverOpen(!isPopoverOpen);
+    }, [setIsPopoverOpen, isPopoverOpen]);
+    const closePopover = useCallback(() => {
+      setIsPopoverOpen(false);
+    }, [setIsPopoverOpen]);
+    const filterGroupPopoverId = useGeneratedHtmlId({
+      prefix: 'filterGroupPopover',
+    });
+
+    const button = useMemo(
+      () => (
+        <EuiFilterButton
+          iconType="arrowDown"
+          onClick={onButtonClick}
+          isSelected={isPopoverOpen}
+          numFilters={numFilters}
+          hasActiveFilters={hasActiveFilters}
+          numActiveFilters={numActiveFilters}
+        >
+          {filterName.charAt(0).toUpperCase() + filterName.slice(1, filterName.length)}
```

Comment:
Ok. so this does not look right to me and tells me that the value is not i18n'd. We should not be manipulating the value/command name

Reply:
e3209c9d2c95d9afd7c3625d2c09df3052c5cc13